src/main/scala/beam/utils/scenario/urbansim/ParquetScenarioReader.scala

```diff
@@ -94,8 +95,19 @@ object ParquetScenarioReader extends UrbanSimScenarioReader with LazyLogging {
     val personId = getIfNotNull(rec, "person_id").toString
     val householdId = getIfNotNull(rec, "household_id").toString
     val age = getIfNotNull(rec, "age").asInstanceOf[Long].toInt
+    val isFemaleValue = {
+      val value = getIfNotNull(rec, "sex").asInstanceOf[String]
```

Comment:
This will fail in runtime because `sex` is `int64` or `Long` in Java.

Reply:
fixed